documentation/asciidoc/topics/yaml/prometheus_service_monitor.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: prometheus
+  name: infinispan-monitoring # Names the service monitor.
+  namespace: {example_crd_name} # Namespace to install the service monitor.
+spec:
+  endpoints:
+    - targetPort: 11223 # Port where {brandname} exposes the metrics endpoint.
```

Comment:
We can define the `port: infinispan-adm` parameter and don't care about port exposition number

Reply:
Is that something we need to code before we can document?

Reply:
We can change `targetPort` with the port number to the named port `infinispan-adm`